fragments/usage/README.md

```diff
@@ -0,0 +1,20 @@
+# Usage Recommendations Fragment
+
+This fragment describes conditions under which the model publishers believe the model will perform
+as expected. 
+
+While the [Model Training] section gives users a clear picture of the training conditions, the model
+publisher may also have evidence or intuition that the model will also perform well when used in a 
+different context. For instance, a model might have been trained on data from a particular
+geographic region and time period, but based on some combination of intuition, previous experience,
+and informal experimentation, the model publisher may be confident that it will perform similarly in
+a new geographic region. This section is meant to capture that information in a systematic way,
+while also providing publishers the opportunity to describe their recommendations in a more
+free-form style.
+
```

Comment:
A short sentence/paragraph could be added, noting that the section can also provide possible conditions, where the model will not perform.

Reply:
Updated in a829170 to have separate `recommendations` and `cautions` sections.